src/MxpTag.cpp

```diff
@@ -22,15 +22,15 @@
 #include "TMxpTagParser.h"
 
 MxpTagAttribute::MxpTagAttribute(const QString& name, const QString& value)
-: QPair(name, value)
+: QPair<QString, QString>(name, value)
```

Comment:
In Qt6 `QPair` is a template to `std::pair` so we might as well use it directly for both Qt major versions...

Reply:
I tried to do this, but found out that [here](https://github.com/Mudlet/Mudlet/blob/894e5caa63efb1cd3b947ab10959a3fbfcbef32f/src/TMap.cpp#LL1178C10-L1178C10) a `QPair` is written to a `QDataStream`. To keep things more portable I propose staying with `QPair`.

Reply:
Hmm, if QPair is just a std::pair in Qt6, does it mean existing Qt5-written binary files won't be read in Qt6 or vice-versa?

Reply:
Since QPair is mentioned [here](https://doc.qt.io/qt-6/datastreamformat.html) for both Qt5 and Qt6, I'd assume backwards compatibility is maintained.

Reply:
Alright. I think we could switch over to everything but that one case then, being a fan of std-native containers myself. That is outside the scope of this PR though so not required to get the checkmark here.